Makefile

```diff
@@ -136,6 +136,9 @@ test-verbose: ginkgo-set tidy-vendor
 format:
 	gofmt -l -w pkg/ cmd/
 
+golint:
```

Comment:
do we need lint before build?

Reply:
for ex 
`
_build_containerized: format golint
`

Reply:
Yes, but since we have many problems now it will be very annoying!
So, after we fix the issues, we can add lint as a requisite for building...